projects/wave-core-new/src/lib/datasets/add-data/add-data.component.ts

```diff
@@ -47,6 +48,15 @@ export class AddDataComponent implements OnInit {
         };
     }
 
+    static createUploadButton(): AddDataButton {
+        return {
+            name: 'Upload',
+            description: 'Upload data from you local computer',
+            iconSrc: this.createIconDataUrl('Upload'),
```

Comment:
don't we have this method now in `util`? If not, I guess we should have.

Reply:
c4343fc66306430154e75232fe8bae110eb76a4d